doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -146,17 +163,20 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `req.sec.gen.03` | General | The Architecture **must** provide tenants isolation. |
 | `req.sec.gen.01` | General | The Architecture **must** support policy based RBAC. |
-| `req.sec.gen.02` | General | The Architecture shall/must xxx. |
+| `req.sec.gen.02` | General | The Architecture **must** support having a centralised authentication and authorisation mechanism. |
 | `req.sec.gen.03` | General | The Architecture shall/must xxx. |
 | `req.sec.zon.01` | Zoning | The Architecture **must** support identity management (specific roles assigned to a domain or tenant), Storage and password encryption; RBAC for Infrastructure and Tenants; Tenant isolation; authentication management (integration with Corporate Identity Management). |
```

Comment:
Duplicate to several other requirements, e.g. req.sec.gen.03, req.sec.gen.01.

Reply:
@hdamker are you referring to req.sec.gen.02 or req.sec.zon.01?

Reply:
great idea @pgoyal01 . Done